Core/DetectorConstruction.cpp

```diff
@@ -673,6 +704,18 @@ void DetectorConstruction::ConstructTargetRun5()
     transform, xadFilling_logical, "xadFillingGeom",
     fWorldLogical, true, 0, checkOverlaps
   );
+  
+  // Vacuum
+  G4Tubs* smallChamber_vac = new G4Tubs(
+    "smallChamber_vac", 0 * cm, chamber_radius_inner - 0.01 * cm, 
+    halfZ , 0 * degree, 360 * degree);
```

Comment:
Compiler crashed here as halfZ was not defined. Did you mean xadFilling_halfthickness?

Reply:
Sorry I forgot to define this halfZ
const double halfZ = 2.6 * cm;